activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
> If there's mentions of `ActiveSupport::HWIA` in guides or elsewhere we should expand them to the correct name.

No actually there isn't any reference to `ActiveSupport::HWIA` in the guides, I'm just too lazy to write the entire name. :blush:

Should I open another pull request to remove this constant for better references / easier release notes ? Are we waiting @jeremy's and @matthewd's approval on that ?